integration-test/src/test/resources/trino-ci-testset/testsets/jdbc-postgresql/00007_varchar.txt

```diff
@@ -0,0 +1,47 @@
+CREATE SCHEMA
+
+USE
+
+CREATE TABLE
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb01 (
+   id integer,
+   name char(20)
+)
+COMMENT ''"
+
+"CREATE TABLE ""test.gt_postgresql"".varchar_db1.tb02 (
+   id integer,
+   name char(65536)
+)
+COMMENT ''"
+
+<QUERY_FAILED> Unknown type 'char(65537)' for column 'name'
```

Comment:
I think the error message should not be understandable by the user

Reply:
The error message is from Trino itself, not from the Gravitino connector.